cmd/openshift-tests/openshift-tests.go

```diff
@@ -111,6 +112,7 @@ func newRunCommand() *cobra.Command {
 		SilenceErrors: true,
 		RunE: func(cmd *cobra.Command, args []string) error {
 			return mirrorToFile(opt, func() error {
+				fmt.Fprintln(os.Stderr, "openshift-test version:", version.Get().String())
```

Comment:
One more, sorry.  Exclude `dry-run`.

Reply:
with `--dry-run`, I get this on stderr anyway:
```
I0326 18:32:47.828294  369176 test_context.go:419] Tolerating taints "node-role.kubernetes.io/master" when considering if nodes are ready
```

Reply:
Fixed, still I think it's quite useless.